test/Model/loading.js

```diff
@@ -45,4 +45,30 @@ describe('loading', function() {
       });
     });
   });
+
+  describe('unfetch', function() {
+    it('unloads doc after Share doc has nothing pending', function() {
+      var setupModel = this.backend.createModel();
+      var model = this.model;
+      setupModel.add('colors', {id: 'green', hex: '00ff00'}, function(err) {
+        if (err) return done(err);
+
+        model.fetch('colors.green', function(err) {
+          if (err) return done(err);
+          expect(model.get('colors.green.hex')).to.equal('00ff00');
+          // Queue up a pending op.
+          model.set('colors.green.hex', '00ee00');
+          // Unfetch. The pending op prevents the doc from immedialy being unloaded.
+          model.unfetch('colors.green');
+          // Once there's nothing pending on the model/doc...
+          model.whenNothingPending(function() {
+            // Racer doc should be unloaded.
+            expect(model.get('colors.green')).to.equal(undefined);
+            // Share doc should be unloaded too.
+            expect(model.connection.getExisting('colors', 'green')).to.equal(undefined);
```

Comment:
👍 I'm guessing you verified this fails w/o the fix?

Reply:
Yup.

That said, checking it again, I realized I forgot the `done` in the `it` signature, so the failure was being attributed to another test. Fixed that up now!